WordPress/gutenberg
# Update compact search control metrics

Follow-up to https://github.com/WordPress/gutenberg/pull/54548.

## What?
Update `SizeControl` metrics when size is `compact`.

## Why?
Better visual balance. The current spacing works well at 40/48px, but at the smaller 32px size it's a bit much.

## Testing Instructions
1. Run `npm run storybook:dev`, open `SearchControl` and set size to `compact`
2. Observe the updated spacing

### Before
<img width="600" alt="Screenshot 2023-09-20 at 14 34 41" src="https://github.com/WordPress/gutenberg/assets/846565/8320e728-124e-479f-b911-ab10f8288345">


### After
<img width="573" alt="Screenshot 2023-09-20 at 14 41 31" src="https://github.com/WordPress/gutenberg/assets/846565/6f2d4a9e-4076-4a60-830e-097e02123a67">
